core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -10659,4 +10659,46 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
         .withSpark()
         .ok(expectedSparkSql);
   }
+
+  @Test public void newLineInLiteral() {
+    final String query = "SELECT 'netezza\n to bq'";
+    final String expectedBQSql = "SELECT 'netezza to bq'";
+    sql(query)
+        .withBigQuery()
+        .ok(expectedBQSql);
+  }
+
+  @Test public void newLineInWhereClauseLiteral() {
+    final String query = "SELECT *\n"
+        + "FROM \"foodmart\".\"employee\"\n"
+        + "WHERE \"first_name\" ='Maya\n"
+        + " Gutierrez'";
+    final String expectedBQSql = "SELECT *\nFROM foodmart.employee\n"
+        + "WHERE first_name = 'Maya Gutierrez'";
```

Comment:
This do not works in BigQuery. Please check

Reply:
checked ,i changed the formatting and rewrote the expected query

Reply:
Ok